worlds/witness/data/WitnessLogicVariety.txt

```diff
@@ -0,0 +1,1223 @@
+Door - 0x0368A (Stairs) - 0x03677
+158143 - 0x3C125 (Control Room Left) - 0x014E9 - Black/White Squares & Dots & Eraser & Symmetry
+158144 - 0x0367C (Control Room Right) - 0x014E9 - Colored Squares & Dots & Eraser & Stars & Stars + Same Colored Symbol
+159411 - 0x0069D (Ramp EP) - 0x03676 & 0x275FF - True
+159413 - 0x00614 (Lift EP) - 0x275FF & 0x03675 - True
+
+Quarry Boathouse (Quarry Boathouse) - Quarry - True - Quarry Boathouse Upper Front - 0x03852 - Quarry Boathouse Behind Staircase - 0x2769B:
+158146 - 0x034D4 (Intro Left) - True - Stars
+158147 - 0x021D5 (Intro Right) - True - Shapers & Rotated Shapers & Triangles
+158148 - 0x03852 (Ramp Height Control) - 0x034D4 & 0x021D5 - Rotated Shapers
+158166 - 0x17CA6 (Boat Spawn) - True - Boat
+Door - 0x2769B (Dock) - 0x17CA6
+Door - 0x27163 (Dock Invis Barrier) - 0x17CA6
+
+Quarry Boathouse Behind Staircase (Quarry Boathouse) - The Ocean - 0x17CA6:
+
+Quarry Boathouse Upper Front (Quarry Boathouse) - Quarry Boathouse Upper Middle - 0x17C50:
+158149 - 0x021B3 (Front Row 1) - True - Shapers & Eraser
+158150 - 0x021B4 (Front Row 2) - 0x021B3 - Shapers & Eraser
+158151 - 0x021B0 (Front Row 3) - 0x021B4 - Shapers & Eraser
+158152 - 0x021AF (Front Row 4) - 0x021B0 - Shapers & Eraser
+158153 - 0x021AE (Front Row 5) - 0x021AF - Shapers & Eraser
+Door - 0x17C50 (First Barrier) - 0x021AE
+
+Quarry Boathouse Upper Middle (Quarry Boathouse) - Quarry Boathouse Upper Back - 0x03858:
+158154 - 0x03858 (Ramp Horizontal Control) - True - Shapers & Eraser
+159402 - 0x00859 (Moving Ramp EP) - 0x03858 & 0x03852 & 0x3865F - True
+
+Quarry Boathouse Upper Back (Quarry Boathouse) - Quarry Boathouse Upper Middle - 0x3865F:
+158155 - 0x38663 (Second Barrier Panel) - True - True
+Door - 0x3865F (Second Barrier) - 0x38663
+158156 - 0x021B5 (Back First Row 1) - True - Shapers & Negative Shapers & Eraser
+158157 - 0x021B6 (Back First Row 2) - 0x021B5 - Shapers & Negative Shapers & Eraser
+158158 - 0x021B7 (Back First Row 3) - 0x021B6 - Shapers & Negative Shapers & Eraser
+158159 - 0x021BB (Back First Row 4) - 0x021B7 - Shapers & Negative Shapers & Eraser
+158160 - 0x09DB5 (Back First Row 5) - 0x021BB - Shapers & Negative Shapers & Eraser
+158161 - 0x09DB1 (Back First Row 6) - 0x09DB5 - Stars & Stars + Same Colored Symbol & Eraser & Colored Squares
+158162 - 0x3C124 (Back First Row 7) - 0x09DB1 - Stars & Stars + Same Colored Symbol & Eraser & Colored Squares
+158163 - 0x09DB3 (Back First Row 8) - 0x3C124 - Stars & Stars + Same Colored Symbol & Eraser & Colored Squares & Triangles
+158164 - 0x09DB4 (Back First Row 9) - 0x09DB3 - Stars & Stars + Same Colored Symbol & Eraser & Colored Squares & Triangles
+158165 - 0x275FA (Hook Control) - True - Shapers & Eraser
+158167 - 0x0A3CB (Back Second Row 1) - 0x09DB4 - Black/White Squares & Colored Squares & Eraser & Shapers
+158168 - 0x0A3CC (Back Second Row 2) - 0x0A3CB - Stars & Stars + Same Colored Symbol & & Eraser & Shapers
```

Comment:
Typo (it was mine)
```suggestion
158168 - 0x0A3CC (Back Second Row 2) - 0x0A3CB - Stars & Stars + Same Colored Symbol & Eraser & Shapers
```

Reply:
My ass just forgot to `git push` yesterday lmao